cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -130,14 +141,122 @@ impl ToString for LockedManifest {
     }
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[cfg_attr(test, derive(proptest_derive::Arbitrary))]
 pub struct LockedManifestCatalog {
     #[serde(rename = "lockfile-version")]
-    version: Version<1>,
+    pub version: Version<1>,
     /// original manifest that was locked
-    manifest: TypedManifestCatalog,
-    /// locked groups
-    groups: Vec<LockedGroup>,
+    pub manifest: TypedManifestCatalog,
+    /// locked pacakges
+    pub packages: Vec<LockedPackageCatalog>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[cfg_attr(test, derive(proptest_derive::Arbitrary))]
+pub struct LockedPackageCatalog {
+    // region: original fields from the service
+    // These fields are copied from the generated struct.
+    pub attr_path: String,
+    pub broken: bool,
+    pub derivation: String,
+    pub description: String,
+    pub license: String,
+    pub locked_url: String,
+    pub name: String,
+    pub pname: String,
+    pub rev: String,
+    pub rev_count: i64,
+    #[cfg_attr(test, proptest(strategy = "crate::utils::proptest_chrono_strategy()"))]
+    pub rev_date: chrono::DateTime<chrono::offset::Utc>,
+    #[cfg_attr(test, proptest(strategy = "crate::utils::proptest_chrono_strategy()"))]
+    pub scrape_date: chrono::DateTime<chrono::offset::Utc>,
+    pub stabilities: Vec<String>,
+    pub unfree: bool,
+    pub version: String,
+    // endregion
+
+    // region: converted fields
+    pub outputs: BTreeMap<String, String>,
+    pub outputs_to_install: Vec<String>,
```

Comment:
nit: it looks like `outputs_to_install` is just passed through, why is it in this region? Did you convert `outputs` just for convenience? Or backwards compatibility with pkgdb?